NEWS.md

```diff
@@ -1,3 +1,15 @@
+## CHANGES WITH 34:
+
+  * Support temporary AWS credentials
+  * Composer-api and worker-api: OAuth2 support
+  * Install docs in RHEL 8.5 and 9.0 images
+  * RHEL-8.5 / RHEL-9.0: RHSM DNF plugins are now enabled by default on `ec2` and `ami` images
+  * Add support for new osbuild stages required for RHEL EC2 SAP images
+
+Contributions from: Alexander Todorov, Antonio Murdaca, Brian C. Lane, Chloe Kaubisch, Diaa Sami, Gianluca Zuccarelli, Jakub Rusz, Juan Abia, Martin Sehnoutka, Ondřej Budai, Simon Steinbeiss, Thomas Lavocat, Tom Gundersen, Tomas Hozza, Sanne Raymaekers, Yi He
```

Comment:
Can you format this so the line length is <= 80 characters, please? Just so we are consistent. Also line 6 might be an issue.